apps/python/scart.py

```diff
@@ -981,56 +987,58 @@ def usage(exitcode=0):
+        channels = "*"
 
-            if verbose:
-                print(
-                    "Adding stream to list: {}.{}.{}.{} {} - {}".format(
-                        stream[2], stream[3], stream[4], stream[5], stream[0], stream[1]
-                    ),
-                    file=sys.stderr,
-                )
-            archiveIterator.append(
-                stream[0], stream[1], stream[2], stream[3], stream[4], stream[5]
-            )
+# Populate streams for both Dump and Import mode
+streams = []
+if listFile:
+    for stream in readStreamTimeList(listFile):
+        if stream[0] >= stream[1]:
+            print("info: ignoring {}.{}.{}.{} - start {} after end {}"
+                  .format(stream[2], stream[3], stream[4], stream[5],
+                          stream[0], stream[1]), file=sys.stderr)
+            continue
+        streams.append(stream)
 
-    elif nslcFile:
-        print("Stream file: '{}'".format(nslcFile), file=sys.stderr)
-        streams = readStreamList(nslcFile)
-        for stream in streams:
-            if verbose:
-                print(
-                    "Adding stream to list: {}.{}.{}.{} {} - {}".format(
-                        stream[0], stream[1], stream[2], stream[3], tmin, tmax
-                    ),
-                    file=sys.stderr,
-                )
-            archiveIterator.append(
-                tmin, tmax, stream[0], stream[1], stream[2], stream[3]
-            )
+elif nslcFile:
+    for stream in readStreamList(nslcFile):
+        streams.append((tmin, tmax, stream[0], stream[1], stream[2], stream[3]))
 
+elif not checkSDS:
+    if networks == "*":
+        streams.append((tmin, tmax, "*", "*", "*", channels))
     else:
-        if networks == "*":
-            archiveIterator.append(tmin, tmax, "*", "*", "*", channels)
-        else:
-            items = networks.split(",")
-            for n in items:
-                n = n.strip()
-                archiveIterator.append(tmin, tmax, n, "*", "*", channels)
+        items = networks.split(",")
+        for n in items:
+            n = n.strip()
+            nsl = n.split(".")
+            if len(nsl) == 1:
+                streams.append((tmin, tmax, nsl[0], "*", "*", channels))
+            elif len(nsl) == 2:
+                streams.append((tmin, tmax, nsl[0], nsl[1], "*", channels) )
+            elif len(nsl) == 3:
+                streams.append((tmin, tmax, nsl[0], nsl[1], nsl[2], channels))
+            elif len(nsl) == 4:
+                streams.append((tmin, tmax, nsl[0], nsl[1], nsl[2], nsl[3]))
+            else:
+                print("error: wrong format of -n option - stopping",
+                      file=sys.stderr)
+                sys.exit(-1)
 
+streamDict = {}
+if dump:
+    for stream in streams:
+        archiveIterator.append(
+            stream[0], stream[1], stream[2],
+            stream[3], stream[4], stream[5])
+        if verbose:
+            print("Adding stream to list: {}.{}.{}.{} {} - {}"
```

Comment:
f strings, see above

Reply:
I didn't modify the `print` statement, I have just move it around. I would avoid mixing changes to the formatting with functional changes.

Reply:
Fine for me. We tend to correct whenever lines are touched but this is up to you.